package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "zksync-cli",
-  "version": "0.2.6",
+  "version": "0.0.0-development",
```

Comment:
As far as I know it doesn't really matter what version is in package.json, we can even leave 0.2.6, semantic-release will publish new version anyway. I changed it just to avoid any confusion for people who will look into this file.

Reply:
OH ok